test/core/end2end/goaway_server_test.cc

```diff
@@ -207,6 +208,13 @@ static void my_cancel_ares_request(grpc_ares_request* request) {
 }
 
 int main(int argc, char** argv) {
+  // TODO(hork): rewrite this test with a custom EventEngine DNS Resolver
```

Comment:
@yijiem please add your name to this, since it's part of the DNS resolver work.

Reply:
Done.